qolmat/imputations/rpca/rpca_noisy.py

```diff
@@ -1,4 +1,4 @@
-"""Script for an the noisy RPCA."""
+"""Script for the noisy RPCA."""
```

Comment:
It's possible that the `the` should  be dropped instead of `an` -- I can't tell.